app/models/manageiq/providers/openstack/inventory/persister/definitions/storage_collections.rb

```diff
@@ -17,5 +16,22 @@ def initialize_storage_inventory_collections
         end
       end
     end
+    add_cloud_volume_backups
+  end
+
+  def add_cloud_volume_backups(extra_properties = {})
+    add_collection(cloud, :cloud_volume_backups, extra_properties) do |builder|
+      if targeted?
+        builder.add_properties(:parent => manager.cinder_manager)
+        builder.add_default_values(:ems_id => manager.cinder_manager.try(:id))
+      else
+        builder.add_default_values(:ems_id => manager.id)
+      end
+      # targeted refresh workaround-- always refresh the whole backup collection
```

Comment:
I would probably write here why:

``Because OpenStack doesn't give us UUIDs of changed volume_backups, we just get an event that one of them changed```

Btw. is this a bug? :-D It's like super unhelpful event. :-) Or does it use ID we don't recognize?

Reply:
Good call on the improved comment. I'm not sure whether this is officially a bug, but it is a really unhelpful event. There are some other IDs present like the request ID, but nothing that helps us as far as I know.